torchaudio/models/rnnt.py

```diff
@@ -377,12 +377,20 @@ class _Joiner(torch.nn.Module):
     Args:
         input_dim (int): source and target input dimension.
         output_dim (int): output dimension.
+        joiner_activation (str, optional): activation function to use in the joiner
+            Must be one of ("relu", "tanh"). (Default: "relu")
+
     """
 
-    def __init__(self, input_dim: int, output_dim: int) -> None:
+    def __init__(self, input_dim: int, output_dim: int, joiner_activation: str = "relu") -> None:
```

Comment:
nit: since the class is called `_Joiner`, we can just call the argument `activation` for simplicity